app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include RenderConditionConcern
+
+    check_or_render_not_found -> { enabled? }
+
+    VERIFICATION_OPTIONS = {
+      ipp: 'ipp',
+      remote: 'remote',
+    }
```

Comment:
If we're going to use an enum-ish thing like this I think we should freeze it

```suggestion
    }.freeze
```